Documentation/Security/GuidelinesAdministrators/DatabaseAccess.rst

```diff
@@ -1,94 +1,135 @@
+For example, do not commit files such as:
 
-Database not within web document root with SQLite
-=================================================
+-   :file:`.env`
+-   :file:`config/system/settings.php` (if it contains credentials directly)
 
-If using SQLite as underlying database, a database is stored in a single
-file. In TYPO3, its default location is the :ref:`var/sqlite <Environment-var-path>` path
-of the instance which is derived from environment variable :php:`TYPO3_PATH_APP`. If that
-variable is **not** set which is often the case in not Composer based instances, **the database
-file will end up in the web server accessible document root directory :file:`typo3conf/`**!
-In such a setup it is important to configure Web servers to not deliver :file:`.sqlite` files.
+Exposing credentials publicly, or even within internal team repositories,
+creates unnecessary risk.
 
+**Recommendation:** Use environment-specific configuration and exclude
+credential files from version control using mechanisms like :file:`.gitignore`.
 
-Disallow external access
-========================
+For more information and best practices, see:
+:ref:`Avoid storing credentials in version control <t3coreapi:version-control-credentials>`.
 
-The database server should only be reachable from the server that your
-TYPO3 installation is running on. Make sure to disable any access from
-outside of your server or network (settings in firewall rules) and/or
-do not bind the database server to a network interface.
+..  _security-database-mysql-access:
 
-If you are using MySQL, read the chapter
-`Server Options <https://dev.mysql.com/doc/refman/8.0/en/server-option-variable-reference.html>`_
-in the manual and check for the "skip-networking" and "bind-address" options
-in particular.
+Use strong passwords and limit privileges
+=========================================
 
+When using MySQL, database users must authenticate before connecting.
+Permissions are granted at various levels (for example, per database,
+per table, or per action such as SELECT or INSERT).
 
-.. index::
+**Best practices:**
+
+-   Use a secure password for the TYPO3 database user. See
+    :ref:`secure password guidelines <security-secure-passwords>`.
+-   Do **not** use obvious usernames like `root`, `admin`, or `typo3`.
+-   Create a dedicated user with **access only to the TYPO3 database**, and only
+    with the permissions it requires (SELECT, INSERT, UPDATE, DELETE, etc.).
+-   Avoid granting administrative privileges such as `LOCK TABLES`, `FILE`,
+    `PROCESS`, `RELOAD`, or `SHUTDOWN` unless absolutely necessary.
+
+..  index::
+    Database; SQLite
+    pair: Security guidelines; SQLite
+
+..  _security-database-sqlite:
+
+Keep SQLite files out of the web root
+=====================================
+
+SQLite stores the database in a single file. By default, TYPO3 places this file
+in the :ref:`var/sqlite <Environment-var-path>` directory, derived from the
```

Comment:
Hi Hi. In the backup documentation you wrote, it's not necessary needed to backup "var/" ;-)

Reply:
I also wrote that you should not backup a database by back upping its files? So what is you point here?